src/components/Hoverable/index.js

```diff
@@ -34,9 +49,14 @@ class Hoverable extends Component {
      * @param {Boolean} isHovered - Whether or not this component is hovered.
      */
     setIsHovered(isHovered) {
-        if (isHovered !== this.state.isHovered) {
+        if (isHovered !== this.state.isHovered && !(isHovered && this.hoverDisabled)) {
             this.setState({isHovered}, isHovered ? this.props.onHoverIn : this.props.onHoverOut);
         }
+
+        // we reset the Hover block in case touchmove was not first after touctstart
```

Comment:
Can you help me understand the race condition that can occur here?

Reply:
I think it is possible that the touchmove never fires.